mlflow/models/docker_utils.py

```diff
@@ -40,6 +40,9 @@
 {install_mlflow}
 
 {custom_setup_steps}
+
```

Comment:
Can you add a note here explaining why this is needed for future viewers of this command?
i.e. 
`# granting read/write access and conditional execution authority to all child directories and files to allow for deployment to <insert name of AWS serverless management service here with a link to the service's main page>` or something along those lines.